### PR TITLE
create monoalphabetic cipher

### PR DESCRIPTION
**Pull request by radadiyamohit81**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: ciphers/mono_alphabetic_ciphers.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+
+def translate_message(key, message, mode):
+    """
+    >>> translate_message("QWERTYUIOPASDFGHJKLZXCVBNM", "When you do the common things in life in an uncommon way, you will command the attention of the world", "encrypt")
```
Comment (cclauss) [inline ciphers/mono_alphabetic_ciphers.py:6]:
```suggestion
    >>> translate_message(
    ...     "QWERTYUIOPASDFGHJKLZXCVBNM", 
    ...     "When you do the common things in life in an uncommon way, "
    ...     "you will command the attention of the world", "encrypt")
```
The automated tests hate lines longer than 88 characters.

File: ciphers/mono_alphabetic_ciphers.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+def decrypt_message(key, message):
+    """
+    >>> decrypt_message("QWERTYUIOPASDFGHJKLZXCVBNM", "When you do the common things in life in an uncommon way, you will command the attention of the world")
+    'Bpcy fig mi epc vizziy ephyol hy shnc hy ky gyvizziy bkf, fig bhss vizzkym epc keecyehiy in epc bidsm'
+    """
+    return translate_message(key, message, "decrypt")
+
+
+def main():
+    myMessage = "When you do the common things in life in an uncommon way, you will command the attention of the world"
```
Comment (cclauss) [inline ciphers/mono_alphabetic_ciphers.py:48]:
```suggestion
    myMessage = (
        "When you do the common things in life in an uncommon way, "
        "you will command the attention of the world"
    )
```

File: ciphers/mono_alphabetic_ciphers.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    >>> translate_message("QWERTYUIOPASDFGHJKLZXCVBNM", "When you do the common things in life in an uncommon way, you will command the attention of the world", "encrypt")
+    'Vitf ngx rg zit egddgf zioful of soyt of qf xfegddgf vqn, ngx voss egddqfr zit qzztfzogf gy zit vgksr'
+    """
+    translated = ""
+    charsA = LETTERS
+    charsB = key
+    if mode == "decrypt":
+        # For decrypting, we can use the same code as encrypting. We
+        # just need to swap where the key and LETTERS strings are used.
+        charsA, charsB = charsB, charsA
```
Comment (cclauss) [inline ciphers/mono_alphabetic_ciphers.py:15]:
```suggestion
    charsA = LETTERS if mode == "decrypt" else key
    charsB = key if mode == "decrypt" else LETTERS
    translated = ""
```

**Comment by cclauss** [comment]
Can you please do `python3 -m pip install flake8` on your local computer and then.
`flake8 mono_alphabetic_ciphers.py`

**Comment by radadiyamohit81** [comment]
let me try it

**Comment by radadiyamohit81** [comment]
Thank you @cclauss, I learned new things from this Thank you so much

File: ciphers/mono_alphabetic_ciphers.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    myKey = "QWERTYUIOPASDFGHJKLZXCVBNM"
+    myMode = "decrypt"  # set to 'encrypt' or 'decrypt'
+
+    if myMode == "encrypt":
+        translated = encrypt_message(myKey, myMessage)
+    elif myMode == "decrypt":
+        translated = decrypt_message(myKey, myMessage)
+    print("Using key %s" % (myKey))
+    print("The %sed message is:" % (myMode))
+    print(translated)
```
Comment (cclauss) [inline ciphers/mono_alphabetic_ciphers.py:54]: f- strings

Comment (radadiyamohit81) [reply ciphers/mono_alphabetic_ciphers.py:54]: Ya I will doing...

File: ciphers/mono_alphabetic_ciphers.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+def main():
+    myMessage = "Hello World"
+    myKey = "QWERTYUIOPASDFGHJKLZXCVBNM"
+    myMode = "decrypt"  # set to 'encrypt' or 'decrypt'
+
+    if myMode == "encrypt":
+        translated = encrypt_message(myKey, myMessage)
+    elif myMode == "decrypt":
+        translated = decrypt_message(myKey, myMessage)
+    print(f"Using key %s {myKey}")
```
Comment (cclauss) [inline ciphers/mono_alphabetic_ciphers.py:52]: %s ??

Comment (radadiyamohit81) [reply ciphers/mono_alphabetic_ciphers.py:52]: yes next update to resolve it.

File: ciphers/mono_alphabetic_ciphers.py [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    myMessage = "Hello World"
+    myKey = "QWERTYUIOPASDFGHJKLZXCVBNM"
+    myMode = "decrypt"  # set to 'encrypt' or 'decrypt'
+
+    if myMode == "encrypt":
+        translated = encrypt_message(myKey, myMessage)
+    elif myMode == "decrypt":
+        translated = decrypt_message(myKey, myMessage)
+    print(f"Using key is {myKey}")
+    print(f"The {myMode}ed message is: {translated}")
```
Comment (cclauss) [inline ciphers/mono_alphabetic_ciphers.py:53]:
```suggestion
    print(f"Using the key {myKey}, the {myMode}ed message is: {translated}")
```

**Comment by cclauss** [comment]
One last change needed...
`charsA, charsB, symIndex, myMessage, myKey, myMode` -->
`chars_a, chars_b, sym_index, message, key, mode`

**Review by cclauss — APPROVED** [review]
Cool...  I would have simplified with [`from string import ascii_letters`](https://docs.python.org/3/library/string.html) but this way works too.

**Comment by radadiyamohit81** [comment]
Ya its works too and Thank you so much

**Pull request merged**